docs/guides/development/cli.md

```diff
@@ -157,6 +157,38 @@ npx coda execute path/to/pack.ts Items --dynamicUrl=https://example.com/api/tabl
 ```
 
 
+### Running metadata functions
+
+More complicated Packs may have a lot of code outside of the core `execute` function, used to generate metadata like parameter autocomplete values, dynamic schemas, etc. You can run those functions using the CLI as well, using the same `execute` command but passing in additional information in name of the formula or sync table.
+
+```sh
+# Parameter autocomplete
+npx coda execute <formula name>:autocomplete:<paramName> [query] [params JSON]
```

Comment:
What do you think about including a specific example here, that shows what "params JSON" actually looks like on the CLI? I think it can get confusing to know what's being asked for in terms of quoting and the like.

And then if we're doing concrete examples, maybe one that explains, hey in most cases don't even worry about the JSON, it's only relevant if your implementation looks at other parameters, and otherwise you don't even have to pass a value for that.

Reply:
Done. Also rearranged the content a bit, PTAL.